external/anomaly/anomaly_classification/configs/configuration.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from sys import maxsize
+
+from attr import attrs
+from ote_sdk.configuration import ConfigurableParameters
+from ote_sdk.configuration.elements import (ParameterGroup,
+                                            add_parameter_group,
+                                            configurable_integer,
+                                            selectable,
+                                            string_attribute)
+from ote_sdk.configuration.model_lifecycle import ModelLifecycle
+
+from anomaly_classification.configs.configuration_enums import POTQuantizationPreset
+
+
+@attrs
+class AnomalyClassificationConfig(ConfigurableParameters):
```

Comment:
I think this should be named as `BaseAnomalyClassificationConfig`. Whichever inherits this class should be `AnomalyClassificationConfig`